pkg/controller/user-cluster-controller-manager/util/pause.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
```

Comment:
```
~/DEV/loodse/src/k8c.io/kubermatic(master)
$ ack 'package util' | wc -l
      16
```

* Please, not another purposeless name
* IMHO, even with better name this doesn't belong to _user-cluster-controller-manager_

Reply:
I actually struggle to come up with a better place and name :-D Also I intended for this to indeed belong to the UCCM and not be reused by other controller-managers. Maybe `common` would be a better package name. I'm open for suggestions.

Reply:
Deal!